index.html

```diff
@@ -0,0 +1,110 @@
+  <input type="email" id="email" name="email" placeholder="Email" required><br><br>
+     <label id="number-label" >Age:</label>
+  <input type="number" id="number" name="age" min="10" max="50" placeholder="Age"><br><br>
+          
+     <label id="number-label" >Which option best describes your current role?:</label>
+ <select id="dropdown" name="role" class="form-control" required>
+        <option disabled selected value>Select current role</option>
+        <option value="student">Student</option>
+        <option value="job">Full Time Job</option>
+        <option value="learner">Full Time Learner</option>
+        <option value="preferNo">Prefer not to say</option>
+        <option value="other">Other</option>
+      </select><br><br>   
+   <p>Would you recommend freeCodeCamp to a friend?</p>
+      <label>
+        <input
+          name="user-recommend"
+          value="definitely"
+          type="radio"
+          class="input-radio"
+          checked/>Definitely</label>
+      <label>
+        <input
+          name="user-recommend"
+          value="maybe"
+          type="radio"
+          class="input-radio"/>Maybe</label>
+
+      <label><input
+          name="user-recommend"
+          value="not-sure"
+          type="radio"
+          class="input-radio"/>Not sure</label>  
+          
+             <p>
+        What would you like to see improved?
+        <span class="clue">(Check all that apply)</span>
+      </p>
+
+      <label><input
+          name="prefer"
+          value="front-end-projects"
+          type="checkbox"
+          class="input-checkbox"/>Front-end Projects</label >
+    
+     
+      <label><input
+          name="prefer"
+          value="challenges"
+          type="checkbox"
+          class="input-checkbox"/>Challenges</label>
+      <label><input
+          name="prefer"
+          value="open-source-community"
+          type="checkbox"
+          class="input-checkbox"/>Open Source Community</label>
+      <label><input
+          name="prefer"
+          value="gitter-help-rooms"
+          type="checkbox"
+          class="input-checkbox"/>Gitter help rooms</label>
+      <label><input
+          name="prefer"
+          value="videos"
+          type="checkbox"
+          class="input-checkbox"/>Videos</label>
+      <label><input
+          name="prefer"
+          value="city-meetups"
+          type="checkbox"
+          class="input-checkbox"/>City Meetups</label >
+      <label><input
+          name="prefer"
+          value="wiki"
+          type="checkbox"
+          class="input-checkbox"/>Wiki</label>
+      
+      
+       <p>Any comments or suggestions?</p>
+      <textarea
+        id="comments"
+        class="input-textarea"
+        name="comment"
+        placeholder="Enter your comment here..."></textarea>
+          <button type="submit" id="submit" >
+        Submit
+      </button><br><br>
+        </form>
+      </div>
+    </div> 
```

Comment:
I would recommend  to the div tag be to changed to a section tag for semantic tags

Reply:
Great job with your reviews. They are gentle and you included a reason.
- [ ] _(optional)_  Hi. Please I suggest giving your code a proper indentation. By doing so, your code will be easier to understand for you and for your collaborators